r_basics_transform_data/r_basics_transform_data.md

```diff
@@ -364,7 +372,9 @@ This filter statement will return a data frame that only contains the 4th row, i
 
 ![On the left, the first few columns of the `covid_testing data frame: mrn, first_name, and last_name`.  The first three rows are labeled "False", while the forth row is labeled "True" and colored blue, and there, the mrn value matches the one we're trying to match.  This data frame is followed by an arrow pointing to a new data frame on the right, which only has the one selected / blue row, the one that had the matching mrn value.](media/filter_covid_example.png)<!--style = "max-width:1000px;"-->
 
-<div class = "warning">
+<div class = "help">
```

Comment:
this div box also kinda feels too big to me, although I don't feel as incredibly strongly about it as the one below. 

It seems to contain important instructional info that isn't just troubleshooting, though, ie, what the difference between = and == is. My rationale for maybe adjusting things is that we wouldn't want to see someone skim or skip over this, since they aren't encountering any issues right now, only to not get that crucial bit of info.

Reply:
I think this is a repeat of the previous highlight box to section comment